### DISCUSSION TITLE
Unable to successfully authenticate

### DISCUSSION THREAD

**Discussion by LIUZiqi1996** [community/community#144319] [GitHub Education]
### Hi everyone,

I am a PhD student at the Sorbonne University. I used my registration certificate or my student card to submit for certification, but it kept failing. I also changed my name to the same as my real name, but it still didn't work. Is there any way to solve this?

Below is my registration certificate and the rejection email
![微信图片_20241112170334](https://github.com/user-attachments/assets/feedc64e-587f-44d4-87dd-8475131c862b)
![Attestation_scolarit](https://github.com/user-attachments/assets/32464bd0-df50-4ffe-88da-a3e6f796782a)

**Comment by huddmax** [comment]
Hi @LIUZiqi1996,

It sounds frustrating to go through this process and not have it work! Here are a few steps that might help:

1. **Double-check the Documents**: Make sure the registration certificate or student card shows your full name exactly as it appears in your GitHub account. The document should clearly display your student status and dates for the current semester or academic year.

2. **File Format and Quality**: Sometimes, GitHub’s system may reject documents if they’re unclear or not in the correct format. Ensure the files you’re uploading are clear, high-resolution images or PDFs.

3. **Use a University Email**: If possible, try submitting your application using your university-issued email (e.g., ending in .edu or your university’s domain). This can often help with verification.

4. **Contact GitHub Education Support**: If you’ve tried all the above steps and it’s still not working, it may help to contact GitHub Education support directly. You can explain the situation and attach the documents in your message. Sometimes, a manual review can resolve issues like this.

I hope this helps, and good luck with your verification!

**Comment by huddmax** [comment]
I also had issues with the verification process and had to submit my application four times before it was approved. What finally helped me was using my university email and uploading clear photos of my documents taken directly from my phone.

Hopefully, this approach can work for you as well!

**Reply by LIUZiqi1996** [reply]
Also regarding the name, my current name passed the last certification, why did it fail the second time? I don’t understand.
![微信图片_20241113104414](https://github.com/user-attachments/assets/fb697d06-42f7-4125-9df9-8225e8721e88)

**Comment by LIUZiqi1996** [comment]
I downloaded my official registration certificate directly from the adum
website, and then converted it into an image, so there shouldn't be any
clarity issues.

Hudson Machado ***@***.***> 于2024年11月12日周二 19:25写道：
